backend/otel/otel.go

```diff
@@ -324,7 +324,7 @@ func (o *Handler) HandleTrace(w http.ResponseWriter, r *http.Request) {
 				keyedErrorMessages[sessionID] = append(keyedErrorMessages[sessionID], &kafkaqueue.Message{
 					Type: kafkaqueue.PushBackendPayload,
 					PushBackendPayload: &kafkaqueue.PushBackendPayloadArgs{
-						ProjectVerboseID: &projectID,
```

Comment:
just making sure i understand this bug:

- `projectID` on 314 is an integer initialized once for the whole loop
- `projectID` set to first value in the map
- this line puts the memory address of the variable in the message
- next iteration of the loop updates `projectID` to be second value in the map
1st message and 2nd message have the same pointer here with the same value, rather than the two different project ids